addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -283,7 +293,7 @@ def test_discount_100_percent(self):
         """ Create an invoice with a 100% discount """
         with freeze_time(self.frozen_today), \
                 patch(f"{self.certificate_module}.fields.datetime.now", lambda x=None: self.frozen_today), \
-                patch(f"{self.certificate_module}.sha1", lambda x: sha1()):
+                patch(f"{self.move_module}.sha1", lambda x: sha1()):
```

Comment:
A little handmade contextmanager would be more elegant but that's not your job to do that :disappointed: